examples/maps_heatmaps/src/index.ts

```diff
@@ -126,471 +120,204 @@ function getPoints() {
+  [37.778833, -122.414932],[37.778834, -122.414898], [37.77874, -122.414757],[37.778501, -122.414433],[37.778182, -122.414026],
+  [37.777851, -122.413623],[37.777486, -122.413166],[37.777109, -122.412674],[37.776743, -122.412186],[37.77644, -122.4118],[37.776295, -122.411614],[37.776158, -122.41144],[37.775806, -122.410997],[37.775422, -122.410484],
+  [37.775126, -122.410087],[37.775012, -122.409854],[37.775164, -122.409573],[37.775498, -122.40918],[37.775868, -122.40873],
+  [37.776256, -122.40824],[37.776519, -122.407928],[37.776539, -122.407904],[37.776595, -122.407854],[37.776853, -122.407547],[37.777234, -122.407087],
+  [37.777644, -122.406558],[37.778066, -122.406017],[37.778468, -122.405499],[37.778866, -122.404995], [37.779295, -122.404455], [37.779695, -122.40395],[37.779982, -122.403584],[37.764826, -122.424922],
+  [37.764796, -122.425375], [37.764782, -122.425869], [37.764768, -122.426089],[37.764766, -122.426117],
+  [37.764723, -122.426276],[37.764681, -122.426649],[37.782012, -122.4042], [37.781574, -122.404911], [37.781055, -122.405597],
+  [37.780479, -122.406341], [37.779996, -122.406939],[37.779459, -122.407613], [37.778953, -122.408228],[37.778409, -122.408839],[37.777842, -122.409501],
+  [37.777334, -122.410181],[37.776809, -122.410836],[37.77624, -122.411514],[37.775725, -122.412145],[37.77519, -122.412805],[37.774672, -122.413464],[37.774084, -122.414186],
+  [37.773533, -122.413636],[37.773021, -122.413009],[37.772501, -122.412371],[37.771964, -122.411681],[37.771479, -122.411078],[37.770992, -122.410477],[37.770467, -122.409801],
+  [37.77009, -122.408904],[37.769657, -122.408103],[37.769132, -122.407276],[37.768564, -122.406469],[37.76798, -122.405745],[37.76738, -122.405299],[37.766604, -122.405297],[37.765838, -122.4052],
+  [37.765139, -122.405139],[37.764457, -122.405094],[37.763716, -122.405142],[37.762932, -122.405398],[37.762126, -122.405813],[37.761344, -122.406215],
+  [37.760556, -122.406495],[37.759732, -122.406484],[37.75891, -122.406228],[37.758182, -122.405695],[37.757676, -122.405118],[37.757039, -122.404346],
+  [37.756335, -122.403719],[37.755503, -122.403406],[37.754665, -122.403242],[37.753837, -122.403172],[37.752986, -122.403112],[37.751266, -122.403355]] ;  
+
+//addAllToDatabase();  //calls the function that adds all the data
+
+//generate random date for the info an the date- day/month/year
+function randomDate()
+{
+  let day,month,year:number;
+  day=Math.floor(Math.random() * 29) + 1;
+  month=Math.floor(Math.random() * 12) + 1
+  year=Math.floor(Math.random() * 30) + 1990
+  return [day,month,year]
+}
+
+// add Single doc into database
+function addNewImage(label:string,lat: number, lng:number, year: number, month: number, day:number, url:string){
```

Comment:
if you want to keep functionality to add data to the DB, please move it to another file (e.g. test_data_utils.ts), as this is just for debugging and should not be in the main file.